requirements.txt

```diff
@@ -1,7 +1,6 @@
-numpy==1.26.0
 pyxdg
 coloredlogs
-pandas<=1.5
+pandas==2.2.3
```

Comment:
What was your motivation for pinning pandas to a precise version?

(I'm a tad concerned that strict requirements like this might cause conflicts if `bibeasy` is installed into an environment alongside other scientific packages, similar to Julien's `teaching` venv.)

Generally Python recommends [wide dependency ranges](https://packaging.python.org/en/latest/discussions/install-requires-vs-requirements/#requirements-files) for packages on PyPI, with more strict dependencies used when freezing an existing environment for reproducibility purposes:

> Whereas [project installation] requirements are minimal, [frozen requirements] often contain an exhaustive listing of pinned versions for the purpose of achieving [repeatable installations](https://pip.pypa.io/en/latest/topics/repeatable-installs/#repeatability) of a complete environment.
> _((Aside: ignore the fact that we use a "requirements file" for our "install_requires"; hence my edits to the quote aha.))_